client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -73,62 +68,66 @@ const ManageBookings = (): JSX.Element => {
     return component;
   };
 
+  let currentBookings: (JSX.Element | undefined)[];
+  const currentBookingRequests = requests.filter(
+    (request) => request.start > new Date() && request !== selectedBooking,
+  );
+  if (currentBookingRequests.length) {
+    currentBookings = currentBookingRequests.map((request) => (
+      <Booking
+        key={`${request.sitterId}${request.start.toString()}`}
+        status={request.status}
+        start={request.start}
+        end={request.end}
+        sitterId={request.sitterId}
+      />
+    ));
+  } else {
+    currentBookings = [<NoBooking key="noPastBooking" text="No current bookings found" />];
+  }
```

Comment:
I think it's a bit unnecessary to return an array here. We can wrap the above mapping in a `React.Fragment` (`<></>`) and return just a single JSX element. That way we can get rid of the artificial array brackets `[ ]` in the `else` case too. Ditto throughout.